aten/src/ATen/native/cuda/AveragePool2d.cu

```diff
@@ -326,24 +406,29 @@ Tensor& avg_pool2d_backward_out_cuda_template(
     outputHeight, outputWidth);
 
   gradInput.resize_as_(input);
+  gradInput.unsafeGetTensorImpl()->empty_tensor_restride(memory_format);
```

Comment:
NITPICK: same here as with the restride comment above.

Reply:
fixed